JSTests/microbenchmarks/to-hex.js

```diff
@@ -0,0 +1,8 @@
+//@ requireOptions("--useUint8ArrayBase64Methods=1")
+
+let buffer = new Uint8Array(16 * 1024);
+for (let i = 0; i < buffer.length; ++i)
+    buffer[i] = i & 0xff;
+
+for (let i = 0; i < 1e4; ++i)
+    buffer.toHex();
```

Comment:
This might be a non-issue, but is there anyway to ensure that future optimizations don't completely remove the call since the value is unused?

Reply:
Fixed.